pydis_site/apps/content/resources/guides/python-guides/discordpy_help_command.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Custom Help Command
+description: "Overwrite discord.py's help command to implement custom functionality"
+---
+
+# Custom Help Command
+
+
+ First,  a [basic walkthrough](https://gist.github.com/InterStella0/b78488fb28cadf279dfd3164b9f0cf96) by Stella#2000 on subclassing the HelpCommand will provide some foundational knowledge required before attempting a more customizable help command.
+
+---
+
+## Custom Subclass of Help Command
+If this does not fit your needs and you require a more customizable help command, you can subclass HelpCommand and add individual command details. Below is a basic demonstration:
+
+```python
+class MyHelpCommand(commands.HelpCommand):
+    async def command_callback(self, ctx, *,command=None):
+        if command:
+            await ctx.send(f"This is the help page for the command {command} ")
+        else:
+            await ctx.send("This is the front page for the bots help command")
+bot.help_command = MyHelpCommand()
+```
+---
+You can handle when there is no command and make a fancy embed;  a page that describes the bot and shows a list of commands is usually ideal here, however if a command is passed in, you can get display detailed information of the command with reference from the following documentations below:
```

Comment:
This language might be a bit confusing to readers, "You can handle when there is no command" is trying to explain if no command is passed in and just the bare help command, however the language itself seems to be describing something else. 
```suggestion
You can handle when there is no command passed into the help command and make a fancy embed;  a page that describes the bot and shows a list of commands is usually ideal here, however if a command is passed in, you can get display detailed information of the command with reference from the following documentations below:
```

Reply:
Thank you for the reviews, I have adjusted the code based on your feedback so I believe everything should be good now.